js/index.js

```diff
@@ -1 +1,97 @@
 // Your code goes here
+const nav = document.querySelectorAll('nav a');
+const funbus = document.querySelector('.intro img');
+const rickRoll = document.querySelector('#ashly');
+const drag = document.querySelector('#drag ');
+const right = document.querySelector('#right-click ');
+const car1 = document.querySelector('#car1 ');
+const car2 = document.querySelector('#car2 ');
+const car3 = document.querySelector('#car3 ');
+const opera = document.querySelector('#Opera ');
+const green = document.querySelector('#green');
+const bubble1 = document.querySelector('#bubble1');
+const bubble2 = document.querySelector('#bubble2');
+
+//picture click event and function #############################
+
```

Comment:
Nice job being creative with your event listeners. I like that you did audio event listeners